staging/src/k8s.io/code-generator/cmd/defaulter-gen/output_tests/wholepkg/zz_generated.defaults.go

```diff
@@ -0,0 +1,85 @@
+//go:build !ignore_autogenerated
+// +build !ignore_autogenerated
+
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by defaulter-gen. DO NOT EDIT.
+
+package wholepkg
+
+import (
+	runtime "k8s.io/apimachinery/pkg/runtime"
```

Comment:
pretty sure this is what is picking up the `k8s.io/apimachinery` dependency, not sure if that's intentional

Reply:
If this were named testdata, I think it would be OK?  Will try

Reply:
Actually no.  Sigh.  These tests used to be triggered by `go generate` in the gengo repo, and are now covered by verify-codegen.  Will restore the publishing bot part.

Reply:
next push